astropy/samp/integrated_client.py

```diff
@@ -429,7 +429,7 @@ def ereply(self, msg_id, status, result=None, error=None):
         >>> from astropy.samp import SAMPIntegratedClient, SAMP_STATUS_ERROR
         >>> cli = SAMPIntegratedClient()
         >>> ...
-        >>> cli.ereply("abd", SAMP_STATUS_ERROR, result={},
+        >>> cli.ereply("message_id", SAMP_STATUS_ERROR, result={},
```

Comment:
#17720 would add a `# codespell:ignore` instruction here, but it's better not to clutter docstrings with such instructions.

Reply:
agreed !